validator/client/runner.go

```diff
@@ -83,24 +83,24 @@ func run(ctx context.Context, v Validator) {
 				wg.Add(1)
 				go func(role pb.ValidatorRole, id string) {
 					defer wg.Done()
+					tpk := id
+					if len(id) > 16 {
+						tpk = id[:16]
+					}
+					log := log.WithFields(logrus.Fields{
+						"pubKey": "0x" + tpk,
```

Comment:
for better coherence with the rest of our logs can we print this as a hex ? `tpk` is encoded as a string

Reply:
As per above `tpk` is a hex string so this should have the same format as the rest of the entries.